src/styles/styles/index.js

```diff
@@ -0,0 +1,9 @@
+import {Platform} from 'react-native';
+
+const styles = Platform.select({
+    ios: require('./styles.ios').default,
+    default: require('./styles.native').default,
+    android: require('./styles.android').default,
+});
+
+export default styles;
```

Comment:
This file is not needed

Reply:
@ArekChr it's actually needed cause when I removed it, the platform-specific styles are not picked, maybe I am missing something here, mind pointing me on how to fix this thanks 🙏

Reply:
It should work if you remove styles.native

Reply:
There might be conflict between ios/android and native file

Reply:
Okay so the way styles are imported on the app is via `[path]/styles/styles.js`
So removing index.js files means I have to update `styles.native.js` to `index.js` and `styles.ios.js` to index.ios.js and lastley `styles.android.js` to index.android.js

And since iOS and Android still need to extend the global styles we have to import the index.js into iOS and Android which would lead to Require cycles and throw errors.

<img width="326" alt="Screenshot 2023-02-22 at 10 32 26 AM" src="https://user-images.githubusercontent.com/33870502/220579759-83ef00ac-3432-40f8-92e8-a73811c81455.png">
<img width="564" alt="Screenshot 2023-02-22 at 10 35 52 AM" src="https://user-images.githubusercontent.com/33870502/220580560-de08cb99-5d4a-4f23-8a54-8795aaf9eba5.png">

Reply:
Require cycle makes styles undefined, please fix that